wordpress-mobile/AztecEditor-iOS
# Release 1.4.4

Release version 1.4.4

To test:
`gutenberg-mobile`: https://github.com/wordpress-mobile/gutenberg-mobile/pull/721
`WPiOS`: https://github.com/wordpress-mobile/WordPress-iOS/pull/11215

CI should pass after the formal pod releases.
